pkg/controller/kieapp/defaults/merge.go

```diff
@@ -131,42 +132,42 @@ func getRoleBindingReferenceSlice(objects []rbacv1.RoleBinding) []v1.OpenShiftOb
 func mergeDeploymentConfigs(baseline []appsv1.DeploymentConfig, overwrite []appsv1.DeploymentConfig) []appsv1.DeploymentConfig {
 	if len(overwrite) == 0 {
 		return baseline
-	} else if len(baseline) == 0 {
+	}
+	if len(baseline) == 0 {
 		return overwrite
-	} else {
-		baselineRefs := getDeploymentConfigReferenceSlice(baseline)
-		overwriteRefs := getDeploymentConfigReferenceSlice(overwrite)
-		for overwriteIndex := range overwrite {
-			overwriteItem := overwrite[overwriteIndex]
-			baselineIndex, _ := findOpenShiftObject(&overwriteItem, baselineRefs)
-			if baselineIndex >= 0 {
-				baselineItem := baseline[baselineIndex]
-				err := mergeLabels(&overwriteItem.ObjectMeta, &baselineItem.ObjectMeta) //reverse merge to maintain changes
-				if err != nil {
-					logrus.Errorf("%v", err)
-					return nil
-				}
-				err = mergo.Merge(&overwriteItem.ObjectMeta, baselineItem.ObjectMeta)
-				if err != nil {
-					logrus.Errorf("%v", err)
-					return nil
-				}
-				mergedSpec, err := mergeSpec(baselineItem.Spec, overwriteItem.Spec)
-				if err != nil {
-					logrus.Errorf("%v", err)
-					return nil
-				}
-				overwriteItem.Spec = mergedSpec
+	}
+	baselineRefs := getDeploymentConfigReferenceSlice(baseline)
+	overwriteRefs := getDeploymentConfigReferenceSlice(overwrite)
+	for _, overwriteItem := range overwrite {
+		baselineIndex, _ := findOpenShiftObject(&overwriteItem, baselineRefs)
+		if baselineIndex >= 0 {
+			baselineItem := baseline[baselineIndex]
+			err := mergeLabels(&overwriteItem.ObjectMeta, &baselineItem.ObjectMeta) //reverse merge to maintain changes
+			if err != nil {
+				logrus.Errorf("%v", err)
+				return nil
 			}
+			err = mergo.Merge(&overwriteItem.ObjectMeta, baselineItem.ObjectMeta)
+			if err != nil {
+				logrus.Errorf("%v", err)
+				return nil
+			}
+			mergedSpec, err := mergeSpec(baselineItem.Spec, overwriteItem.Spec)
+			if err != nil {
+				logrus.Errorf("%v", err)
+				return nil
+			}
+			overwriteItem.Spec = mergedSpec
 		}
-		slice := make([]appsv1.DeploymentConfig, combinedSize(baselineRefs, overwriteRefs))
-		err := mergeObjects(baselineRefs, overwriteRefs, slice)
-		if err != nil {
-			logrus.Errorf("%v", err)
-			return nil
-		}
-		return slice
 	}
+	slice := make([]appsv1.DeploymentConfig, combinedSize(baselineRefs, overwriteRefs))
+	err := mergeObjects(baselineRefs, overwriteRefs, slice)
+	if err != nil {
+		logrus.Errorf("%v", err)
+		return nil
+	}
+	return slice
+
```

Comment:
I keep looking at this, which looks like 100+ lines of diff, but I think is just a small cosmetic change to prevent an extra indent. Please revert so it's clear what's changed and what hasn't. We can have a pure refactor commit later on and include this.